src/CompositionRoot.ts

```diff
@@ -62,7 +67,8 @@ function getCompositionRoot(repositories: Repositories) {
         getWithOptions: new GetEntityWithOptionsUseCase(repositories),
         save: new SaveEntityUseCase(
```

Comment:
better to add directly `repositories` and in the use case get them like for example in the use case `GetDiseaseOutbreakByIdUseCase`